src/service/okapi.service.ts

```diff
@@ -30,58 +30,48 @@ class OkapiService extends RestService {
         if (!!response && response?.statusCode >= 200 && response?.statusCode <= 299) {
           let accessToken;
           let refreshToken;
-          const headers = !!response?.headers ? response?.headers : undefined;
 
-          if (!!headers['set-cookie']) {
-            const matchAccess = /folioAccessToken=([^;\s]*)/gi;
-            const matchRefresh = /folioRefreshToken=([^;\s]*)/gi;
-            const cookies = !!headers['set-cookie'] ? headers['set-cookie'] : undefined;
-            let foundAccess;
-            let foundRefresh;
+          const headers = response?.headers;
 
-            if (!!cookies && Array.isArray(cookies)) {
-              let matched;
+          if (headers?.['set-cookie'] && Array.isArray(headers['set-cookie'])) {
```

Comment:
```suggestion
          if (!!headers?.['set-cookie'] && Array.isArray(headers['set-cookie'])) {
```
This is Javascript, we need to use the double negate to enforce boolean.

Reply:
Yeah, I thought so as well but it does not pass default linting.

Reply:
Oh, that condition is already occurring with `const headers = response?.headers;`

Reply:
Nope. It is for headers undefined. Anyway, I will revert to double negation.

Reply:
```js
var struct = { header: "header" };
var header = struct?.header;

console.log(typeof header);
console.log(typeof !!header);
```

Produces:
```
 "string"
 "boolean"
```

We want to ensure that the comparison is a boolean comparison.